src/main/java/edu/harvard/iq/dataverse/util/BundleUtil.java

```diff
@@ -28,18 +29,15 @@ public static String getStringFromBundle(String key, List<String> arguments) {
         return getStringFromBundle(key, arguments, bundle);
     }
 
-    public static String getStringFromBundle(String key, List<String> arguments, ResourceBundle bundle) {
+    public static String getStringFromBundleNoMissingCheck(String key, List<String> arguments, ResourceBundle bundle) throws MissingResourceException {
```

Comment:
```suggestion
    private static String getStringFromBundleNoMissingCheck(String key, List<String> arguments, ResourceBundle bundle) throws MissingResourceException {
```

In addition to making the new method above private, we could do the same for `getResourceBundle` below.